src/pages/MarketPlacePage/MarketPlace.tsx

```diff
@@ -1,21 +1,41 @@
+import { useState } from "react";
 import JobCard from "../../components/JobCard/JobCard";
+import CategoryDropdown from "../../components/CategoryDropdown/CategoryDropdown";
+import projectData from "../../data/dummy_data_extended.json";
+
+interface Job {
+  id: number;
+  name: string;
+  category: string;
+  job_tags: string[];
```

Comment:
hopefully you guys can start fixing these cases at an earlier stage. the cases are a tech debt that's accumulating across all the code.